cmd/mkcgo/generate.go

```diff
@@ -56,10 +69,13 @@ func generateGo(src *mkcgo.Source, w io.Writer) {
 		fmt.Fprintf(w, "}\n\n")
 	}
 
-	typedefs := make(map[string]string, len(src.TypeDefs))
-	for _, def := range src.TypeDefs {
-		typedefs[def.Name] = def.Type
-	}
+	// Generate error wrapper noescape function, which hides the
+	// error state pointer from the Go garbage collector.
```

Comment:
Maybe link another use of this to make it obvious that it's a known way to do this and it's not something that Go will get wise to. 😄 (Could alternatively `linkname` it?)

```suggestion
	// error state pointer from the Go garbage collector.
	// An instance of https://github.com/golang/go/blob/d704ef76068eb7da15520b08dc7df98f45f85ffa/src/runtime/stubs.go#L194-L201
```

Reply:
> (Could alternatively linkname it?)

The `linkname` directive is locked down, better avoid it: https://github.com/golang/go/issues/67401.